# Header must contain time stamp
Header header

# Target configuration for kinematic controller
float32[] configuration